grpc/src/main/java/com/replica/util/RouterConverters.java

```diff
@@ -312,6 +309,16 @@ private static List<String> fetchStreetLegStableIds(Trip.WalkLeg leg) {
                 .collect(toList());
     }
 
+    public static CustomStreetLeg createEmptyCustomStreetLeg(org.locationtech.jts.geom.Point point, Date departureTime,
+                                                             String travelSegmentType, String mode) {
+        // Set departureLocation, instructions, and details params as either "" or null (these values aren't used in
```

Comment:
(as an FYI, I could in theory have created the relevant ~empty versions of objects for these params to the new `WalkLeg` we create, but it's clunky + takes a lot of code. Given that we never access or use these fields when generating proto objects to return to the client, setting them as empty/`null` seemed like a better solution